example/storage-provider-secrets/01-aws-s3-storage-secret.yaml

```diff
@@ -12,30 +12,3 @@ data:
 # s3ForcePathStyle: # set to `true` for S3 compliant providers
 # insecureSkipVerify: # set to `true` for S3 private deployments to skip certificate check
 # trustedCaCert: # set a trusted Certificate authority for private deployments like minio
-
-#### OR ####
```

Comment:
I would suggest if you are removing it from examples then you can remove it from doc as well: https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md#various-ways-to-pass-credentials for all storage providers.

Reply:
Good catch! Thanks for the suggestion.

I'll remove it from the mentioned doc as well.